src/server/api/terrain.js

```diff
@@ -39,10 +39,14 @@ export const handler = ({ method, pathname, headers }) => {
             req
         )
             .then((apiResponse) => {
+                res.set(apiResponse.headers);
                 res.status(apiResponse.status);
                 res.send(apiResponse.data);
             })
-            .catch((e) => {
+            .catch(async (err) => {
+                console.log(err);
```

Comment:
this should `logger.error`.

Reply:
This should be fixed, thanks for the heads up.